pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllSamples.sql

```diff
@@ -78,7 +78,7 @@ LEFT JOIN Institute it ON la.instituteId = it.instituteId
 LEFT JOIN SampleStock ss ON sai.sampleId = ss.sampleId
 LEFT JOIN SampleSlide slide ON slide.sampleId = s.sampleId
 LEFT JOIN Stain stain ON stain.stainId = slide.stain
-LEFT JOIN SampleLCMTube lcm ON lcm.sampleId = s.sampleId
+LEFT JOIN SampleTissuePiece piece ON piece.sampleId = s.sampleId
```

Comment:
Pinery SampleType is going to change to "Tissue Piece" for all LCM tubes and curls. I don't think this will hurt analysis, but not sure?

Reply:
I'm not sure either. I think it won't be visible via sample provenance, so it shouldn't matter for most of analysis. ETL will definitely notice the change, but I don't think it matters.